datadreamer/pipelines/generate_dataset_from_scratch.py

```diff
@@ -56,14 +56,12 @@ def parse_args():
     parser.add_argument(
         "--save_dir",
         type=str,
-        default="generated_dataset",
         help="Directory to save generated images and annotations",
     )
 
     parser.add_argument(
         "--task",
         type=str,
-        default="detection",
```

Comment:
Default values are removed here because they must be set in the utils/config.py file, and we don't want to duplicate them, right?

Reply:
Yes, exactly!